nodejs/week1/homework/src/backend/app.js

```diff
@@ -1,11 +1,35 @@
-const express = require("express");
+const express = require('express');
 const app = express();
 
-// import data here
-const meals = require("./data/meals");
 
-app.get("/", async (request, response) => {
-    response.send("asd");
+const reservations = require('./data/reservations');
+
+const fetch = require('./fetchData');
```

Comment:
It was not necessary to modularize the logic in `./fetchData`, but it is of course ok :)